src/Tizen.Network.Stc/Tizen.Network.Stc/StcRule.cs

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+
+namespace Tizen.Network.Stc
+{
+    /// <summary>
+    /// A class for managing the Stc Rules to match applications.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class StcRule : IDisposable
```

Comment:
Make as a pure c# object

Reply:
``` c#
    public class StcRule
    {
        public DateTime? From { get; set; }
        public DateTime? To { get; set; }

        public NetworkInterface? InterfaceType { get; set; }
        public TimePeriodType? TimePeriod { get; set; }
    }

```

Reply:
IntPtr ConvertToNativeRule(StcRule2 rule)
        {
            Interop.Stc.SafeRuleHandle handle;
            Interop.Stc.Rule.Create(StcManagerImpl.Instance.GetSafeHandle(), out handle);
            if (rule.From.HasValue && rule.To.HasValue)
            {
                Interop.Stc.Rule.SetTimeInterval(_ruleHandle, rule.From.Value, rule.To.Value);
            }
            if (rule.InterfaceType.HasValue)
            {
                Interop.Stc.Rule.SetInterfaceType(_ruleHandle, (NativeNetworkInterface)rule.InterfaceType.Value);
            }
            if (rule.TimePeriod.HasValue)
            {
                Interop.Stc.Rule.SetTimePeriod(_ruleHandle, (NativeTimePeriodType)rule.TimePeriod.Value);
            }
            return handle;
        }

Reply:
Changes were done. Thank you.